README.md

```diff
@@ -298,6 +299,78 @@ expect(queryByText('Hello World')).toHaveStyle([{ color: 'black' }, { fontWeight
 expect(queryByText('Hello World')).not.toHaveStyle({ color: 'white' });
 ```
 
+### `toBeVisisble`
```

Comment:
```suggestion
### `toBeVisible`
```
It would appear each occurrence of "visible" is misspelled in this file. It would create a lot of unnecessary noise for me to call out each one, but we should likely fix the spelling of each misspelling for this file.

Reply:
Oh, my bad, I spelled it wrong and copy/paste it all over the file 😅 , I'll fix it as soon as possible, thanks for spotting it 🙇 .

Reply:
I fixed this in https://github.com/testing-library/jest-native/commit/96c6ac525e1e547540fbeb9b27bc8056d214cb95.